appinventor/components-ios/src/Sprite.swift

```diff
@@ -403,7 +403,30 @@ open class Sprite: ViewComponent, UIGestureRecognizerDelegate {
     updateY(y: y)
     registerChanges()
   }
-  
+
+  /**
+   * Moves the sprite to given point after checking whether the coordinates are numbers.
+   */
+  @objc open func MoveToPoint(_ coordinates: YailList<AnyObject>) {
+    let xCoord = CoerceToDouble(coordinates[1] as AnyObject)
+    let yCoord = CoerceToDouble(coordinates[2] as AnyObject)
+    self.MoveTo(xCoord, yCoord)
```

Comment:
Try testing this with a list of either 0 or 1 elements and make sure that it doesn't crash the app.

Reply:
worked with 1 element!

provided following error in Xcode (using simulator) when I tested with 0 elements: 

"AICompanionApp[9185:9006870] Exception occurred in YAIL: RuntimeError RuntimeError. Irritants: (The arguments  are the wrong number of arguments for MoveToPoint Wrong number of arguments forMoveToPoint)"

App didn't crash from this though.